addon/styles/addon.css

```diff
@@ -38,3 +38,32 @@
 .concept-scheme-multi-select-checkboxes__content {
   width: 100%;
 }
+
+/* SFB-115 */
```

Comment:
Nitpick, but this Jira id has little value as a comment IMO 😄, it's probably already mentioned in the PR where this commit is linked to.